### ISSUE TITLE
Getting synced lyrics from Lyricfind

### ISSUE THREAD

**Issue by swanux** [QuickLyric/QuickLyric#262]
I'm trying to implement something similar to your synced lyrics feature (on desktop, so it wont be a competition). I've looked everywhere on how to use Lyricfind's resources (APIs, scraping, modules), but I didn't find anything. How do you do this?

**Comment by geecko86** [comment]
We're not using their database.

**Comment by swanux** [comment]
The application itself says that lyrics is provided by Lyricfind. If that isn't the case (despite what the app says), how do you manage to acquire synced lyrics?

**Comment by geecko86** [comment]
The rights to display the lyrics are provided by Lyricfind. Our partnership with them allow us to display lyrics breaking copyright law.

We have our own database. The QuickLyric app for Android is open source, and I'm happy to answer any question about it. However I'm not comfortable answering questions about how we operate our business, including how we manage our lyrics database.

**Comment by swanux** [comment]
Okay, I've got it. And excuse me, of course I don't want to ask sensitive questions about how you operate your database.
I'm just interested in how do you generate synced lyrics. (manually, with a neural network or something else) If it's not a proper question, I'd really appreciate some information regarding how did you build up the database (only non-sensitive information of course).

**Comment by JustACuteGirlLookingForAManToFixHerBugs** [comment]
I guess someone is just too busy cash-grabbing to answer... Sad.

**Comment by swanux** [comment]
If anyone is still interested in generating synced lyrics, I've created a repo which contains it in .srt format and it's open for everyone. (I've also described how I generate synced lyrics). Note, that this repo is extremely small (~50 tracks), but it's open so everyone can contribute to it / use it. [Here](https://github.com/swanux/htidal_db) it is. Just a collection. Accessible to everyone, no business behind it @JustACuteGirlLookingForAManToFixHerBugs
